### DISCUSSION TITLE
Raspberry Pi 4

### DISCUSSION THREAD

**Discussion by jt7005** [awawa-dev/HyperHDR#733] [Q&A] [answered]
Hi, Quick question, I have HyperHDR running on a Raspberry Pi 3 and using Hyperspi. Everything is running fine, I now have the option to upgrade to a Raspberry Pi 4 4Gb. Is it worth me upgrading? The main difference I see is that the Raspberry Pi 4 has USB 3 so it would be better for the MS2130 grabber that I am using. So do I buy the Raspberry Pi 4 or just leave things as they are.

**Comment by awawa-dev** [comment, marked as answer]
Hi
Two main advantages: USB3.0 support, which means much faster communication with the grabber (some capture modes can be reserved only for USB3.0) + much faster memory bus, which is crucial if we use LUTs (and we do if HDR tona mapping is turned on and even in SDR to support SDR YUV modes).

Whether you notice the difference is another matter and I can't help you here: as they say, if it ain't broke, don't fix it ;) BTW some time ago, the RPi4 2GB and 1GB reappeared, that much memory is enough for HyperHDR if only this application is to run on Raspberry Pi Lite version of OS.
